salt/utils/error.py

```diff
@@ -3,7 +3,7 @@
 Utilities to enable exception reraising across the master commands
 
 '''
-from __future__ import absolute_import
+from __future__ import absolute_import, unicode_literals
```

Comment:
We should have `print_function` imported as well in all of these files (see [here](https://gist.github.com/terminalmage/7440c8d6e46ed4ee2f69c834d5363512#print_function-and-unicode_literals)).